osquery/tables/system/windows/programs.cpp

```diff
@@ -110,6 +307,16 @@ QueryData genPrograms(QueryContext& context) {
     keyEnumPrograms(k, processedPrograms, results);
   }
 
+  std::set<std::string> userMsixKeys;
```

Comment:
MSIX applications are always installed at the user scope, even if the package was originally provisioned system-wide via `Add-AppxProvisionedPackage` or `DISM`. Provisioning an app only stages the app so that it is automatically installed for new user profiles at login. Once a user logs in, the app is registered under
`HKEY_USERS\<SID>\Software\Classes\Local Settings\Software\Microsoft\Windows\CurrentVersion\AppModel\Repository\Packages` and this location contains all the data we want to pickup with osquery.

While there is system-level store of MSIX applications: `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Appx\AppxAllUserStore`  
These keys do not really contain any additional information we want to grab with osquery.